### PR TITLE
Add info for migrating from local packages to modules

### PR DESCRIPTION
**Pull request by reohjs**
This PR adds a few tips to the 'Recommended change: use modules' section of the 'Migrating to Meteor 1.3' article on converting an app-local packages structure to a modules-based structure.

Ref: https://forums.meteor.com/t/how-to-migrating-from-all-in-packages-to-1-3-modules/20681/3

### CODE REVIEW

File: content/1.3-migration.md [hunk 1]
```diff
@@ -42,8 +42,26 @@ import { EJSON } from 'meteor/ejson';
...
-- If you are using app-local packages to control load order and write unit tests for your application, we recommend you switch to using modules. You can read about our recommended structure for applications and modules in the [Application Structure article](structure.html) of the Meteor Guide, and how to test them in the [Testing article](testing.html).
+- If you are using app-local packages to control load order and write unit tests for your application, we recommend you switch to using modules:
+ - Remove code related to the [Package API](http://docs.meteor.com/#/full/packagejs) from the `package.js` files and rename them to `index.js`, 
+ - Move your local packages to the `imports/` directory.
+ - Add the necessary import statements to each of the modules in your packages.
 
+```js
+api.use('meteor');
+// Turns into
+import { Meteor } from 'meteor/meteor';
```
Comment (stubailo) [inline content/1.3-migration.md]: I don't think there is any reason to have your `index.js` import these dependencies, right?

File: content/1.3-migration.md [hunk 2]
```diff
@@ -42,8 +42,26 @@ import { EJSON } from 'meteor/ejson';
...
+
+api.addFiles('file.js');
+// Turns into
+import './file.js';
+
+// imports/main.js
+// Don't forget to import the package!
+import './local_package/index';
```
Comment (stubailo) [inline content/1.3-migration.md]: This is a bit confusing - this is saying you should have a file called `imports/main.js`, and that should import all of your local packages? That doesn't seem completely right to me. I'd rather when people want to use their module, they import:

```js
import { myModule } from '/imports/myModule'`;
```

**Comment by stubailo** [comment]
I'd also suggest talking about exports in here.

File: content/1.3-migration.md (hunk 1 above)
Comment (reohjs) [inline content/1.3-migration.md]: Right, meant to remove this.

**Comment by reohjs** [comment]
I've updated with suggestions

**Comment by stubailo** [comment]
This is definitely an improvement, thanks!

**Pull request merged**